src/librustc_error_codes/error_codes/E0432.md

```diff
@@ -12,21 +12,59 @@ prefixes, respectively. Also verify that you didn't misspell the import
 name and that the import exists in the module from where you tried to
 import it. Example:
 
+In Rust 2015, paths in `use` statements are relative to the crate root. To
+import items relative to the current and parent modules, use the `self::` and
+`super::` prefixes, respectively.
+
+<<<<<<< Updated upstream
```

Comment:
Is this intentional?

Reply:
No, that's a fail on my side. I'll remove it and squash